CMakeLists.txt

```diff
@@ -326,9 +326,14 @@ if(QUDA_MPI OR QUDA_QMP)
     message(STATUS "Found environment variable MPICC. Using it for MPI detection: $ENV{MPICC}")
     set(MPI_C_COMPILER $ENV{MPICC})
     set(mpimessage True)
+  endif()
+  if((NOT MPI_FORT_COMPILER) AND DEFINED ENV{MPIFORT})
+    message(STATUS "Found environment variable MPIFORT. Using it for MPI detection: $ENV{MPIFORT}")
+    set(MPI_C_COMPILER $ENV{MPICC})
```

Comment:
This is not what we want (MPI_C_COMPILER) - but I am thinking about dropping support for that anyway.

Reply:
Fixed in #820